### PR TITLE
Rly strategy v1

### PR DESCRIPTION
**Pull request by rally-ray**
Updated RLY Strategy to use a contract for calculating participation in LP pools.

The goal of the strategy should look at Balancer/Uniswap LP tokens owned and then those staked in the liquidity mining pools, to account for those RLY holders (in addition to any RLY directly in a wallet now). The strategy uses quadratic weighting to all votes as preferred over raw 1:1 voting power for now. 

This request submitted on behalf of community discussion in RLY forum, proposals, and Discord.

### CODE REVIEW

**Review by gokaiorg — APPROVED** [review]

File: spaces/rally/index.json [hunk 1]
```diff
@@ -1,17 +1,8 @@
 {
+  "token": "0xf1f955016ecbcd7321c7266bccfb96c68ea5e49b",
```
Comment (bonustrack) [inline spaces/rally/index.json:2]: Can you remove this? We don't use this field anymore.

Comment (rally-ray) [reply spaces/rally/index.json]: Done. I think this was a copy/paste error.

**Comment by gokaiorg** [comment]
Your old proposals will not be visible anymore under your space, are you sure you want to go forward with this?

**Comment by amit-rally** [comment]
> Your old proposals will not be visible anymore under your space, are you sure you want to go forward with this?

can you confirm which old proposals will not be visible? is it all of them / expected behavior with a strategy change or something specific to the strategy change being made? block at which the contract being used for the 'contract-call' was deployed perhaps?

**Comment by gokaiorg** [comment]
Yes all current proposals will no longer be visible. The space will be empty. 
This is the normal behaviour when changing the address of strategies.

**Comment by bonustrack** [comment]
@rally-ray @gokaiorg Actually this is not true, if you change strategies your previous proposals will still be visible but the result will be using the new strategies.

**Comment by gokaiorg** [comment]
@bonustrack Ok on the other hand during my local tests we lose absolutely all the proposals of the display.

**Comment by rally-ray** [comment]
Thanks guys, let's go ahead and give it a shot. We have reviewed the JS and don't see anything that looks like it would remove previous entries, and we have taken a catalog of the past proposals for record just in case. Please go ahead and merge the PR.

**Pull request merged**